### ISSUE TITLE
Send2cgeo support for Opencaching

### ISSUE THREAD

**Issue by azhag-pl** [cgeo/cgeo#5554]
Please add support for Opencaching sites in send2cgeo service.

**Comment by rsudev** [comment]
Hi @azhag-pl, there is already some level of support (see https://raw.githubusercontent.com/cgeo/send2cgeo/master/send2cgeoOc.user.js for oc.de and https://raw.githubusercontent.com/cgeo/send2cgeo/master/send2cgeoOcPl.user.js for oc.pl/oc.nl but as we didn't get around to enough testing, we did not release it publicly. But you are welcome to use is and report back bugs.

**Comment by azhag-pl** [comment]
Thanks for the reply. I made some quick testing.

The start was a bit bumpy, as I'm not the kind of guy who actually reads manual, but since I did and learned how send2cgeo actually works, I must say it works like a charm. Both as a userscript and by directly entering https://send2.cgeo.org/add.html?cache=OP1234 in browser. I'm gonna ask for some more testing at OC.pl forum just to be sure.

BTW, the userscript runs at Opera too.

However, I'm talking with OC.pl folks about implementing the feature directly in OC.pl (and therefore in other nodes that share the code), and they're willing to do it if it works with OC. So it appears no userscript and addon is going to be needed for send2cgeo to work.
